content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx

```diff
@@ -234,6 +240,40 @@ const users = await prisma.user.findMany({
 
 See also: [Case sensitivity](case-sensitivity)
 
+### Filtering FAQs
+
+#### How does filtering work at the database level?
+
+For MySQL and PostgreSQL, Prisma Client utilizes the [`LIKE`](https://www.w3schools.com/sql/sql_like.asp) (and [`ILIKE`](https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-LIKE)) operator to search for a given pattern. The operators have built-in pattern matching using symbols unique to `LIKE`. The pattern-matching symbols include `%` for zero or more characters (similar to `*` in other regex implementations) and `_` for one character (similar to `.`)
+
+To match the literal characters, `%` or `_`, make sure you escape those characters. For example:
+
+<!-- prettier-ignore-start -->
+```ts
+const users = await prisma.user.findMany({
```

Comment:
What is the difference between the 2 queries? They seem to have the same filter 😅

Reply:
oops!